test/resources/case9_templates/case9-test-policy.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: policy.open-cluster-management.io/v1
```

Comment:
Could you please also add a managed cluster template to ensure it does not get modified?